### PR TITLE
Dimension description

### PR DESCRIPTION
**Pull request by ptbrowne**
- feat: Add tooltip for dimension which have descriptions
- feat: Use drop-shadow for arrow to be included in shadow
- feat: Add dimension tooltip in data table

### CODE REVIEW

File: app/docs/components.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { markdown } from "catalog";
 
 const Doc = () => markdown`
-> The components used in the User Interface are built upon [rebass](https://rebassjs.org/), a library of React primitive UI components.
+> The components used in the User Interface are built upon [rebass](https://mui.com/), a library of React UI components.
```
Comment (bprusinowski) [inline app/docs/components.tsx]: ...built upon [MUI]...?

Comment (wereHamster) [inline app/docs/components.tsx:4]:
```suggestion
> The components used in the User Interface are built upon [Material-UI](https://mui.com/), a library of React UI components.
```

**Review by bprusinowski — APPROVED** [review]
LGTM!  But I think we are missing the table implementation 😅

**Comment by ptbrowne** [comment]
Fixed the overflow issue and added the table implementation (along with a refactor to try to clean things up, and have better types).

File: app/configurator/components/datatable.tsx [hunk 2]
```diff
@@ -72,8 +75,20 @@ export const PreviewTable = ({
...
+
+  // Tooltip contained inside the table so as not to overflow when table is scrolled
+  const tooltipProps = useMemo(
+    () => ({
+      PopperProps: {
+        container: tooltipContainerRef.current,
+      },
+    }),
+    []
+  );
```
Comment (bprusinowski) [inline app/configurator/components/datatable.tsx:88]: 💯

File: app/charts/table/cell-desktop.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { Box, Theme } from "@mui/material";
 import { makeStyles } from "@mui/styles";
 import { hcl, ScaleLinear } from "d3";
-import * as React from "react";
+import React from "react";
```
Comment (wereHamster) [inline app/charts/table/cell-desktop.tsx:4]: @ptbrowne @bprusinowski https://github.com/facebook/react/pull/18102

**Pull request merged**